docs/Researcher/cli-reference/runai-submit.md

```diff
@@ -357,8 +370,17 @@ runai submit --job-name-prefix -i gcr.io/run-ai-demo/quickstart -g 1
 
 #### -s | --service-type `<string>`
 
-> External access type to interactive jobs. Options are: portforward (deprecated), loadbalancer, nodeport, ingress.
-  
+> External access type to interactive jobs. Options are:
```

Comment:
@itayvallach do all these options correct?

portforward` (deprecated)
    > * `loadbalancer`
    > * `nodeport`
    > * `ingress`
    > * `external-url

Reply:
ingress doesn't exist anymore. should be deleted. the rest are fine.

Reply:
@itayvallach Should we remove the deprecated one as well?

Reply:
no I agree with your below comment. it is still functioning.